noir-projects/aztec-nr/aztec/src/macros/aztec.nr

```diff
@@ -128,31 +122,14 @@ comptime fn generate_contract_library_method_compute_note_hash_and_nullifier() -
     let notes = NOTES.entries();
 
     if notes.len() > 0 {
-        let max_note_packed_len = notes.fold(
-            0,
-            |acc, (_, (_, len, _, _)): (Type, (TypeDefinition, u32, Field, [(Quoted, u32, bool)]))| {
-                if len > acc {
-                    len
-                } else {
-                    acc
-                }
-            },
-        );
-
-        if max_note_packed_len > MAX_NOTE_PACKED_LEN {
-            panic(
-                f"One of the notes has packed len {max_note_packed_len} but the maximum is {MAX_NOTE_PACKED_LEN}",
-            );
-        }
```

Comment:
Moved this check to the note macro itself. This was pretty much a completely incorrect place to have it in.